### ISSUE TITLE
Chart not being rendered as image

### ISSUE THREAD

**Issue by softwarea** [wkhtmltopdf/wkhtmltopdf#5077]
**wkhtmltoimage version(s) affected**: 0.12.6.0 

**OS information**
Windows-10

Hi, I am trying to convert a chart (https://www.highcharts.com) to jpg.

This is my page (just have a look):
https://sidan-live.softwarea.de/test

This is my command at the Windows command prompt:
wkhtmltoimage --enable-javascript --no-stop-slow-scripts https://sidan-live.softwarea.de/test d:\test.jpg

I have also tried to wait for 2 seconds in case the script should be very slow;
wkhtmltoimage --enable-javascript --javascript-delay 2000 https://sidan-live.softwarea.de/test d:\test.jpg

Whatever I tried so far: the output contains the static text "Chart:" (which you can see in the upper left corner of the page), **but no chart.**

Am I doing anything wrong? Or is this a bug? 

Any chance this is related to my page being called via https? Or the included HighCharts script being called via https? Unfortunately I can't go with simple http in my real application. It has to be https.

Any ideas guys? I appreciate any help!!!

Ingmar

**Comment by PhilterPaper** [comment]
Well, highcharts appears to use some pretty heavy duty Javascript, and wkHTMLto* uses a rather back-level rendering engine (WebKit) that does not support up-to-date Javascript, so there's a very good chance that highcharts is trying to do something that is unsupported. You _can_ try removing the animation and pop-up labels and see if that helps, but unfortunately there is probably nothing that can be done about it. You mentioned https... did you try it with http to see if that made any difference? At least that could narrow down where the problem is, although I doubt that's what it is.

**Comment by softwarea** [comment]
Hello Phil, thanks for your feedback. Even though I don't know how wkhtmltoimage works internally, I was afraid of a weak or outdated scripted engine. So, you are probably right: this works for simple scripts only.

Yes, yesterday I was also able to briefly try http (instead of https). But that didn't succeed either. Regardless, it would not be an option for me. I absolutely need https.

So, thanks again for your help, Phil. I see you seem to be doing the support here all by yourself. Although you're not the developer of wkhtmltoimage, are you? Great to see you getting so involved here!

**Comment by PhilterPaper** [comment]
No, I'm not the developer of wkHTMLtoPDF/wkHTMLtoImage, but I do try to help out where I can. I'm busy with my own set of Perl open source: PDF::Builder, PDF::Table, and Text::KnuthPlass.

**Comment by joshuapinter** [comment]
For what it's worth I downgraded the version of Highcharts.js we were using (to 5.0.14 but you may try newer versions) and was able to get it working with wkhtmltopdf just fine. I'm not sure which version of Highcharts stopped working so I would try on this version first and then experiment with upgrading to later versions and post back here how far up you got.
